cdf.js

```diff
@@ -1,6 +1,8 @@
 'use strict';
 
-function cumsum(histogram, offset=0) {
+function cumsum(histogram, offset) {
+  if (offset === undefined) offset = 0;
```

Comment:
Since `cumsum` is only used internally to this module, it might be ever-so-faster to simply pass `0` on line 36 when you don't specify an offset.